### ISSUE TITLE
[bench request]: jqv serving code is now public (Octalab-Inc/jqv) — re-run for a rankable row

### ISSUE THREAD

**Issue by hjmurmur** [fstandhartinger/jevbench#9]
Follow-up to #6 (jqv, partial row in v1.2.7): the serving code is now public, so I would like to request a re-run that can include the held-out hard items on your own hardware, making the row rankable.

**Repository.** https://github.com/Octalab-Inc/jqv — commit `0189b67`. Weights are the public `Qwen/Qwen3-32B` (Apache-2.0, BF16, no fine-tuning). Run instructions written for you: [`docs/jevbench-serving.md`](https://github.com/Octalab-Inc/jqv/blob/0189b67/docs/jevbench-serving.md).

**Configuration.** Identical to the one you measured in #6: `packed` engine (state prefilled once, each question an isolated branch behind a block attention mask, answer read from the option-letter logits in one forward pass, no decoding), one scalar temperature 3.0225814579771493 fitted on 400 MMLU validation items and stored with provenance in the repo (`results/mmlu_packed_qwen3-32b_temperature.json`), served over TypeSafe's wire format (`POST /v1/systemone`), so the `typesafe` adapter runs unchanged with `--key-env ''`. `GET /health` reports `prompt_hash` `4f85a0b34776` and the temperature, which lets you confirm the configuration matches #6 before the run.

```sh
git clone https://github.com/Octalab-Inc/jqv && cd jqv && uv sync
JQV_MODEL=Qwen/Qwen3-32B JQV_ENGINE=packed JQV_TEMPERATURE_FILE=results/mmlu_packed_qwen3-32b_temperature.json \
uv run uvicorn jqv.server:app --host 127.0.0.1 --port 8000
curl -s localhost:8000/health   # model, engine, prompt_hash 4f85a0b34776, calibration.temperature 3.0225814579771493
```

**Hardware.** About 65 GB of memory for the BF16 weights. We ran it on an Apple M5 Max through PyTorch/MPS; on CUDA it should work unchanged (`JQV_DEVICE=cuda`) but we have not run it there. If the masked-attention path gives trouble on your stack, `JQV_ENGINE=kvcache` or `JQV_ENGINE=naive` produce the same choice logits (checked by the equivalence tests in the repo).

**Expected numbers.** On the public tiers the configuration reproduced our own numbers exactly in #6 (easy 1.000, standard 0.958, hard 0.622, hard-tier top-label ECE 0.107), so a healthy run on your side should show the same public-tier figures before the held-out items.

**Cost basis.** Whatever you apply to open weights you run yourselves; the OpenRouter `qwen/qwen3-32b` input tariff you used in #6 is fine with me.

**Disclosure.** Unchanged from #6: no benchmark item was used for training; the only fitted quantity is the temperature (MMLU validation); the public 231 items were used once as a development gate when choosing this configuration over an option-order-averaged variant.

Thank you for the v1.2.7 run and for documenting the endpoint condition; happy to answer questions here.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold

**Comment by fstandhartinger** [comment]
Re-run done on our own hardware from `Octalab-Inc/jqv` 0189b67: `/health` showed prompt_hash 4f85a0b34776 and
temperature 3.0225814579771493 before the run, Qwen/Qwen3-32B BF16, packed engine, on a RunPod H100 NVL (torch 2.11 +
CUDA 12.8), one request at a time. This time all 534 decisions went out, including the 109 held-out hard items, so the
v1.2.7 partial row is replaced by a complete, ranked one.

**#8 of 36, JevBench Score 70.1** (Intelligence 86.1, Calibration 79.0, Speed 74.6, Cost 47.5). Accuracy: easy 100.0 %, standard 95.8 %, judge 92.5 %, hard 64.5 %. Latency p50 0.75 s raw, 1.64 s after the standard self-host adjustment (x2 + 0.15 s); our pod was in Canada, so the raw figure includes the trip from our server in Germany; cost $0.0564 per 1,000 decisions.

The public tiers came out as in #6 (easy 100 %, standard/judge unchanged); on the full hard tier it answered
64.5 %. The earlier tunnel run stays in the history of the repo. Thanks for publishing the serving code.

Page: https://benchmarkheaven.com/jev-models · data and method: https://github.com/fstandhartinger/jevbench/tree/v1.2.8

- Harold

**Comment by hjmurmur** [comment]
Thanks a lot for rerunning jqv from the published code and for completing the full 534-decision benchmark, including the held-out hard set.

It was especially useful to see that the public-tier results reproduced again on your H100 setup, and that the held-out hard accuracy (67.0%) was actually slightly higher than the public hard subset (62.2%). That gives us much more confidence that the public tier has been a reasonable development gate rather than something we accidentally overfit to.

Thanks again for taking the time to reproduce the setup and add jqv to the ranked table.

**Comment by fstandhartinger** [comment]
Congratulations on the JevBench v1.4 release, hjmurmur (Octalab)! Thanks for building and sharing your system.

- jqv: #12 at 44.35

jqv was #6 in v1.3. V1.4 blends in 308 fresh sealed decisions, uses an equal-weight harmonic mean, and applies separate Speed and Cost gates. For your leading row, sealed accuracy was 28.2% versus 80.1% on the public slice. The >25-point gap adjustment applied; the below-50 Intelligence/Cost gates also applied.

Full board and method: https://benchmarkheaven.com/jev-models
